### ISSUE TITLE
Parser rules not checked when minified

### ISSUE THREAD

**Issue by remo5000** [source-academy/cadet-frontend#121]
Expected: An error about only allowing empty list

Got: An array returned. Nothing can be done with the array because of a `no-member` rule.

**Comment by ningyuansg** [comment]
This bug only occurs after minifying the files.

**Comment by ningyuansg** [comment]
Most of the stuff in `src/slang/rules` don't work. Possibly the errors are not being pushed at `src/slang/parser.ts`.

**Comment by ningyuansg** [comment]
![issue121](https://user-images.githubusercontent.com/22320312/41703144-e7f4a7c6-7564-11e8-8ee2-b924fe7f0d84.png)

Seems something went wrong adding stuff to the `walkers` object.

**Comment by ningyuansg** [comment]
Bugged [Scope](https://stackoverflow.com/questions/45506745/what-is-scopes-in-dispatch-of-redux) (minified):
```javascript
0: Closure (Object.keys.forEach.t)
    t: "ArrayExpression"
1: Closure (s.default.forEach.e)
    e: {name: "no-non-empty-list", disableOn: 9, checkers: {…}}
2: Closure
    a: {default: {…}, __esModule: true}
    c: {ErrorType: {…}, ErrorSeverity: {…}, Closure: ƒ, ArrowClosure: ƒ, __esModule: true}
    d: class d
    f: (e,n)=> {…}
    h: class h
    i: {simple: ƒ, ancestor: ƒ, recursive: ƒ, full: ƒ, fullAncestor: ƒ, …}
    l: class l
    m: (n,r)=> {…}
    o: {…}
    p: {Program: ƒ, ExpressionStatement: ƒ, IfStatement: ƒ, FunctionDeclaration: ƒ, VariableDeclaration: ƒ, …}
    r: {…}
    t: {DisallowedConstructError: ƒ, FatalSyntaxError: ƒ, MissingSemicolonError: ƒ, TrailingCommaError: ƒ, freshId: ƒ, …}
    u: class u
3: Global
```

Ok Scope (not minified):
```javascript
0: Closure (compose)
    w1: (node, context) => {…}
    w2: (node, context) => {…}
1: Closure (./src/slang/parser.ts)
    DisallowedConstructError: class DisallowedConstructError
    FatalSyntaxError: class FatalSyntaxError
    MissingSemicolonError: class MissingSemicolonError
    TrailingCommaError: class TrailingCommaError
    compose: ƒ compose(w1, w2)
    createAcornParserOptions: (context) => {…}
    freshId: () => { id++; return 'node_' + id; }
    walkers: {Program: ƒ, ExpressionStatement: ƒ, IfStatement: ƒ, FunctionDeclaration: ƒ, VariableDeclaration: ƒ, …}
    __WEBPACK_IMPORTED_MODULE_0_acorn__: {}
    __WEBPACK_IMPORTED_MODULE_1_acorn_dist_walk__: {simple: ƒ, ancestor: ƒ, recursive: ƒ, full: ƒ, fullAncestor: ƒ, …}
    __WEBPACK_IMPORTED_MODULE_2_common_tags__: {}
    __WEBPACK_IMPORTED_MODULE_4__syntaxTypes__: {a: {…}}
    __WEBPACK_IMPORTED_MODULE_5__types__: {b: ƒ, a: ƒ}
2: Global
```

**Comment by ningyuansg** [comment]
Fix with #126
